### PR TITLE
DOC: Change to matrix.argmax docstring so it reflects what the code does

### PR DESCRIPTION
**Pull request by nikete**
DOC: Fixing docstring so it reflects what the code does

In [222]: x = np.matrix(np.arange(12).reshape((3,4))); x
Out[222]:
matrix([[ 0,  1,  2,  3],
        [ 4,  5,  6,  7],
        [ 8,  9, 10, 11]])

In [223]: x.argmax()
Out[223]: 11

In [224]: x[1,1]=11

In [225]: x.argmax()
Out[225]: 5

### CODE REVIEW

File: numpy/matrixlib/defmatrix.py [hunk 1]
```diff
@@ -783,7 +783,7 @@ def max(self, axis=None, out=None):
 
     def argmax(self, axis=None, out=None):
         """
-        Indices of the maximum values along an axis.
+        Index of the maximum value along an axis, if multiple returns the first one
```
Comment (charris) [inline numpy/matrixlib/defmatrix.py]: Line length <= 75 characters. The first line is also the summary line and stands alone. I'd do something like
```
        Indexes of the maximum values along an axis.

        Return the indexes of the first occurrences of the maximum values
        along the specified axis.  If axis is None, the index is for
        the flattened matrix.
```

**Comment by charris** [comment]
Also, when you do work, always do it in a branch. Your current PR is from your master and the effect is that your master will become out of sync with the numpy repository.

**Comment by charris** [comment]
I updated this in #6970, so closing. Thanks @nikete .

**Comment by nikete** [comment]
Thank you! Sorry for the newb way of going about it.

**Pull request closed without merging**